### DISCUSSION TITLE
How to sign in feather client

### DISCUSSION THREAD

**Discussion by arnavgupta2** [feathersjs/feathers#3055] [Q&A]
### Steps to reproduce

(First please check that this issue is not already solved as [described
here](https://github.com/feathersjs/feathers/blob/dove/.github/contributing.md#report-a-bug) - if it is a general question or suggestion please start a [Discussion](https://github.com/feathersjs/feathers/discussions))

- [ ] Tell us what broke. The more detailed the better.
- [ ] If you can, please create a simple example that reproduces the issue and link to a gist, jrepo, etc. This makes it much easier for us to debug and issues that have a reproducible example will get higher priority.

### Expected behavior

Tell us what should happen

### Actual behavior

Tell us what happens instead

### System configuration

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):

**NodeJS version**:

**Operating System**:

**Browser Version**:

**React Native Version**:

**Module Loader**:

**Comment by daffl** [comment]
This is shown in the [JavaScript frontend guide](https://feathersjs.com/guides/frontend/javascript.html) and documented [in the authentication client API docs](https://feathersjs.com/api/authentication/client.html).
